### PR TITLE
Window creation template scripts - JS/Python

### PR DESCRIPTION
**Pull request by acaetano**
Template developed through [issue 3110](https://github.com/zaproxy/zaproxy/issues/3110), along with its Python port

### CODE REVIEW

File: standalone/window_creation_template.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+//Template for GUI creation
+
+// The following handles differences in printing between Java 7's Rhino JS engine
+// and Java 8's Nashorn JS engine
+if (typeof println == 'undefined') this.println = print;
+
+var absframe = Java.type("org.parosproxy.paros.view.AbstractFrame");
+var tela = Java.extend(absframe);
```
Comment (kingthorin) [inline standalone/window_creation_template.js]: This seems unused

**Comment by acaetano** [comment]
Ok, first and last quoted lines removed, as _absframe_ is still being called in the end.

**Comment by thc202** [comment]
Thanks!

**Comment by acaetano** [comment]
I've just started building an ZAP talk that i plan on submitting to a Brazillian security conference. It addresses some basic concepts about extending ZAP, as most people i've known prefer to use Burp. 

So, expect some pull requests and lots of doubts!

**Comment by thc202** [comment]
Let us know how that goes! :)

**Pull request merged**